### ISSUE TITLE
Safari not Voicing in macOS 12

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#183]
I am now experiencing no Voicing when using Safari.
MacOS 12.6.2
Safari Version 15.6.1 (17613.3.9.1.16)

This happened with a participant last week as well. I could find that issue to re-open it.

**Comment by jessegreenberg** [comment]
OK, thanks for reaching out to them @terracoda, thats unfortunate. We could try to disable Voicing and usages of SpeechSynthesis on that platform so Voicing doesn't seem broken to users. Would that make sense or should we do something else? Or nothing?

**Comment by terracoda** [comment]
What you suggest seems reasonable @jessegreenberg, and I also wanted to share Jame's latest comment...he said:

> Generally speaking, the primary things that ship to older releases are security fixes. Other bugs sometimes make the cut, but usually only if there is wide impact reported immediately after the regression is released, while the release is still current. 
> 
> However, in the interest of improving reliability across all systems, one way you could address this is to file an issue (or submit a patch) to Web Platform Tests. These are shared automation tests that all browser vendors use. If there is a particular Web API that you or a system you maintain relies on, shared automation tests are one of the best ways to ensure it does not regress silently for any browser engine in future releases.
> 
> https://github.com/web-platform-tests/wpt/issues

Is it worth filing a bug report there, as well as blocking Voicing on MacOS 12.x? I am certain is was working before I updated and we have verified Voicing is working on MacOS 13.

**Comment by jessegreenberg** [comment]
OK, thanks. I am going to look into that.

That seems like the right thing to do, but I would want to understand that project a bit more first. I don't think it would be a bug report, it would be more like a feature request to add tests or even make our own pull request for that open source project. Maybe there are already issues open about SpeechSynthesis.

**Comment by terracoda** [comment]
I agree, and I wonder if it something we can look into as part the open-source side of things.

**Comment by jessegreenberg** [comment]
According to https://github.com/phetsims/a11y-research/issues/183#issuecomment-1387244328, Apple is not going to fix this in the 12.x series and so we shouldn't worry about it either.

This hasn't come up in any testing in the past few years, so closing.
